### PR TITLE
Softmax fix in CNN

### PR DESCRIPTION
**Pull request by swghosh**
Fixed softmax activation (added missing `activation` argument to last `Dense` layer) in CNN tutorial.

### CODE REVIEW

**Comment by guoci** [comment]
@swghosh, the loss function is `SparseCategoricalCrossentropy(from_logits=True)`. So, the output layer is in logits and the softmax activation is not required.

**Comment by swghosh** [comment]
Ohh, right @guoci. Sorry, I had overlooked that.
